app/views/measures/_measures.html.erb

```diff
@@ -50,18 +50,33 @@
   <!-- Export tab -->
   <div class="govuk-tabs__panel" id="export">
     <h2 class="govuk-heading-l"><%= measures_heading(anchor: 'export') %></h2>
-    <p>The commodity code for exporting and <%= link_to 'Intrastat reporting', 'https://www.gov.uk/intrastat', target: "_blank" %> is <%= declarable.display_export_code %></p>
+    <p>The commodity code for exporting and <%= link_to 'Intrastat reporting', 'https://www.gov.uk/intrastat', target: "_blank" %> is <%= declarable.display_export_code %>.</p>
 
     <%= render partial: 'declarables/consigned', locals: { declarable: declarable } %>
 
-    <% if declarable.export_measures.any? %>
+    <% if uk_declarable.presence&.export_measures&.any? %>
+      <h3 class="govuk-heading-m" id="uk-export-controls">
+        UK export controls
+      </h3>
       <%= render partial: 'measures/grouped/table', locals: {
-        collection: declarable.export_measures
+        collection: uk_declarable.export_measures
                               .without_supplementary_unit
                               .sort_by(&:key),
         hide_duty_rate: true } %>
+    <% end %>
+
+    <% if TradeTariffFrontend::ServiceChooser.xi? && xi_declarable.presence.export_measures.any? %>
+        <h3 class="govuk-heading-m govuk-!-margin-top-9" id="eu-export-controls">
```

Comment:
4 indentation spaces?

Reply:
I use erb_lint https://github.com/Shopify/erb-lint if its helpful

Reply:
Good point, yeah the VisualStudio has it as well ... I've just installed

Reply:
:tada: 

VScode is awesome